src/app/models/chart-models/chart.ts

```diff
@@ -53,11 +53,13 @@ export class Chart {
   }
 
   get categoricalVariables(): Concept[] {
-    return this.variables.filter( variable => variable.type !== ConceptType.NUMERICAL);
+    return this.variables.filter(variable =>
+      [ConceptType.CATEGORICAL, ConceptType.DATE].indexOf(variable.type) > -1);
```

Comment:
In TypeScript you can use `list.includes(value)` instead of `list.indexOf(value) !== -1`.